features/support/environments.yml

```diff
@@ -3,8 +3,12 @@ SLE_15:
   beta: true
   online_migration_targets:
   - "Basesystem-Module"
+  - "Basesystem-Module"
+  - "Python2-Module"
```

Comment:
Do we need these duplicate lines?

Reply:
Yes, each required module for the migration target product has to be present here to correspond to the complete set of migratoin targets returned - since there is both sp1 and sp2, the modules are listed twice.

I could of course produce a test environment for GA-> SP1 and another for SP1 -> SP2 without duplication, but GA -> SP1 & SP2 is valid, so the initial approach is also valid.